content/tokio/tutorial/streams.md

```diff
@@ -358,6 +358,7 @@ impl Stream for Interval {
             Poll::Ready(_) => {
                 let when = self.delay.when + Duration::from_millis(10);
                 self.delay = Delay {when };
```

Comment:
It'd be nice to fix that missing space too.
```suggestion
                self.delay = Delay { when };
```

Reply:
Fixed!